src/App.tsx

```diff
@@ -1,203 +1,92 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import React, { useEffect, useState } from 'react';
+// import { UserWarning } from './UserWarning';
+// import { USER_ID } from './api/todos';
+import { Header } from './components/Header/Header';
+import { Footer } from './components/Footer/Footer';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import cn from 'classnames';
+import { Errors } from './utils/Errors';
+import { FilterTodosBy } from './utils/FilterTodosBy';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  // kinda useless after the rigestration ?
+  // if (!USER_ID) {
+  //   return <UserWarning />;
+  // }
+
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [hasError, setHasError] = useState<Errors>(Errors.NoError);
+  const [filterBy, setFilterBy] = useState<FilterTodosBy>(FilterTodosBy.All);
+
+  useEffect(() => {
+    setHasError(Errors.NoError);
+    const getTodosData = async () => {
+      try {
+        const response = await getTodos();
+
+        setTodos(response);
+      } catch (error) {
+        setHasError(Errors.UnableToLoad);
+        setTimeout(() => {
+          setHasError(Errors.NoError);
+        }, 3000);
+      }
+    };
+
+    getTodosData();
```

Comment:
```suggestion
    (async () => {
      try {
        const response = await getTodos();

        setTodos(response);
      } catch (error) {
        setHasError(Errors.UnableToLoad);
        setTimeout(() => {
          setHasError(Errors.NoError);
        }, 3000);
      }
    })();
```

You can read more about IIFE - https://developer.mozilla.org/en-US/docs/Glossary/IIFE